src/database/upgrade/9/monitoring_db2.sql

```diff
@@ -0,0 +1,42 @@
+DECLARE GLOBAL TEMPORARY TABLE SESSION.ofTmpStatus (
+    roomID          INTEGER       GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
I don't have a DB2 instance to test this again, but a syntax checker dislikes the `GENERATED ALWAYS AS IDENTITY PRIMARY KEY` part.